### DISCUSSION TITLE
Setup Proxy in root site & not in sub site

### DISCUSSION THREAD

**Discussion by Sharafudeen** [oqtane/oqtane.framework#3356] [General] [answered]
Hi,

I wanted to create two sites which will have a module Module1 with two different behavior.

1) Public site / root site - Module1 can be restricted features for external users who registered using external login. This site (root site) hosted in DMZ server which doesn’t have direct access to the database. This will connect database through proxy server (connection string won’t be exists).

2) Intranet site / sub site - Module1 has all features for internal users. This site (sub site) hosted internally/normal server. This will have direct database connection.

Question: 
1) How do I setup proxy for the public site?
2) How do I skip root site proxy in sub site?
3) How do I design the Module1 for both sites?

If this is not possible in Oqtane then what is an alternate solution?

**Comment by W6HBR** [comment]
This type of configuration is usually handled by proxying the website, not the database server.  I have setup several like this using MS IIS ARR (IIS Application Request Routing) as a reverse proxy.

You have an IIS instance in the DMZ configured to perform the reverse proxy. No site or DB installed here - just proxy functions. Your firewall rules can be configured to only allow TCP/443 from the DMZ IIS proxy to the internal web host.  This way it is only HTTPS traffic from the proxy that traverses the DMZ/firewall.

**Comment by W6HBR** [comment, marked as answer]
Here's an info link to IIS ARR:  https://learn.microsoft.com/en-us/iis/extensions/url-rewrite-module/reverse-proxy-with-url-rewrite-v2-and-application-request-routing
